src/listfunc.c

```diff
@@ -630,6 +630,24 @@ Obj             FuncPOSITION_FIRST_COMPONENT_SORTED (
 
 #include <src/sortbase.h>
 
+// This is a variant of SortDensePlist, which sorts plists by
+// master pointer. It works on non-dense plists, and can be
+// used to efficiently sort lists of small integers
+
+#define SORT_FUNC_NAME SortPlistByMasterPointer
```

Comment:
I am not happy with that name, for two reasons: It does not make it clear at all that one can use it to sort immediates; and "master pointers" really are an implementation detail of GASMAN. Sure, we currently use "fake" master pointers with Boehm GC, too, but to me that's mostly to make it easier to use Boehm GC as a drop-in replacement.

Perhaps `RawSortPlist` or `SortPlistByRawObj` would be less loaded? (Though certainly not great either -- as usual, naming is one of the hardest problems in CS ;-)